app/javascript/src/views/home.vue

```diff
@@ -71,6 +96,9 @@ export default {
       );
       window.location.reload();
     },
+    submitPriceFilter() {
+      this.$store.dispatch('applyPriceFilter', [ this.minPrice, this.maxPrice ]);
```

Comment:
There should be no space after '['               array-bracket-spacing<br>There should be no space before ']'              array-bracket-spacing